lib/services/dialog_import_service.rb

```diff
@@ -113,6 +121,16 @@ def import_from_dialogs(dialogs)
           "resource_actions" => build_resource_actions(dialog)
         )
       )
+      associations.each do |association|
+        association.values.each do |values|
+          values.each do |responder|
+            unless new_or_existing_dialog.dialog_fields.select { |field| field.name == responder }.empty?
```

Comment:
I was hoping we could offload some of this to the database, but after further investigation I don't think that's possible right now.